docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,391 @@
+| media         | image               | default        |                                                                                                      |
+| media         | image               | open_seadragon | This display mode shows only the "Image" file field, formatted as "OpenSeadragon."                      |
+| media         | image               | source         |                                                                                                      |
+| media         | video               | default        |                                                                                                      |
+| media         | video               | source         |                                                                                                      |
+| taxonomy_term | islandora_display   | default        |                                                                                                      |
+| taxonomy_term | islandora_media_use | default        |                                                                                                      |
+| taxonomy_term | islandora_models    | default        |                                                                                                      |
+
+
+### RDF Mappings
+
+The following RDF mappings for the RDF module are included:
+
+| Type          | Bundle              | Comments |
+|---------------|---------------------|----------|
+| node          | islandora_object    |          |
+| media         | audio               |          |
+| media         | document            |          |
+| media         | file                |          |
+| media         | image               |          |
+| media         | video               |          |
+| taxonomy_term | islandora_access    |          |
+| taxonomy_term | islandora_display   |          |
+| taxonomy_term | islandora_media_use |          |
+| taxonomy_term | islandora_models    |          |
+
+
+### Language
+
+These configuration files enable content translation on their bundles. 
+
+| Type          | Bundle              | Comments |
+|---------------|---------------------|----------|
+| node          | islandora_object    | Do not hide untranslatable fields.          |
+| media         | audio               | Do not hide untranslatable fields.          |
+| media         | file                | Do not hide untranslatable fields.          |
+| media         | image               | Do not hide untranslatable fields.          |
+| media         | video               | Do not hide untranslatable fields.          |
+| taxonomy_term | islandora_access    | Language alterable is false.         |
+| taxonomy_term | islandora_media_use | Language alterable is false.         |
+| taxonomy_term | islandora_models    | Language alterable is false.         |
+
+## Views
+
```

Comment:
consider adding link to Glossary:
``(see Glossary: [View](../user-documentation/glossary.md#view))``